index.html

```diff
@@ -27,53 +28,26 @@ <h2 class="splash__title">Instructions</h2>
             <form class="form">
                 <div class="form__input-container">
                     <label class="form__label border-txt" for="column">Columns</label>
-                    <input class="form__input" type="number" name="column" id="column" value="0">
+                    <input class="form__input" type="number" name="column" id="column" placeholder="0" max="12" min="3">
                 </div>
                 <div class="form__input-container">
                     <label class="form__label border-txt" for="row">Rows</label>
-                    <input class="form__input" type="number" name="row" id="row" value="0">
+                    <input class="form__input" type="number" name="row" id="row" placeholder="0" max="12" min="3">
                 </div>
                 <div class="form__input-container">
-                    <label class="form__label border-txt" for="hit">Gophers</label>
-                    <input class="form__input" type="number" name="hit" id="hit" value="0">
+                    <label class="form__label border-txt" for="gopher">Gophers</label>
+                    <input class="form__input" type="number" name="hit" id="gopher" placeholder="0">
+                </div>
+                <div class="error_container">
```

Comment:
I'm finding that this box overlaps the start button when I put in too many gophers. Only seems to be an issue on larger screen sizes

Reply:
Max 72 now

Reply:
Misunderstood, text overlaps the start buttton. Unresolved

Reply:
gave error_container a font-size